app/migrations/20250306124233-adds_project_table.cjs

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.createTable("Project", {
+      project_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        primaryKey: true,
+      },
+      organization_id: {
+        type: Sequelize.UUID,
+        allowNull: false,
+        references: {
+          model: "Organization",
+          key: "organization_id",
+        },
+        onDelete: "CASCADE",
```

Comment:
I'm not sure about these cascades.. we might want to restrict deleting an org that has projects inside

Reply:
In this case we would need an endpoint that handles the full deletion of the org and all of its data once confirmed by the user in the frontend (since that's required for various data safety guidelines).

@isaaccodekill can you add a ticket to create this delete organization endpoint to the epic? Thanks 👍